### DISCUSSION TITLE
New blog post doesn't appear in blog. Please help to fix

### DISCUSSION THREAD

**Discussion by cryptobandteam** [thuliteio/doks#995] [Q&A] [answered]
Please help with answer why new blog post doesn't appear in blog.

I added index.md to content/en/blog/2022-medium-term-report/ but I can't see new blog post

Repo link https://github.com/cryptobandteam/murfalgo-web3-v2

**Comment by h-enk** [comment, marked as answer]
I think you can find your page here: `/blog/say-hello-to-doks/report/`

See also: [Page Bundles](https://gohugo.io/content-management/page-bundles/)

**Reply by cryptobandteam** [reply]
I am getting 404 erorr :(

**Reply by h-enk** [reply]
Better use [Page Bundles](https://gohugo.io/content-management/page-bundles/). So, place `report.md` in the `blog` directory like so: `./content/en/blog/report.md`. 

Also, note that `report.md` has a future date (future posts are not displayed by default): 

```md
date: 2023-11-01T09:19:42+01:00
lastmod: 2023-11-01T09:19:42+01:00
```

So, make it e.g. today:

```md
date: 2023-01-28T09:19:42+01:00
lastmod: 2023-01-28T09:19:42+01:00
```

Or, better create a new blog post like so:

```bash
npm run create blog/my-new.post/index.md
```

Then, to see the post, set `draft: false`

**Reply by cryptobandteam** [reply]
Thank you so much! Everything is worling now :)

**Reply by cryptobandteam** [reply]
Sir, coudl you please also help with instructions to remove top image from the blog post? I see that Images [] is empty, so I can't understand where it comes from in the blog post

---
title: "Say eehello to Doks 👋"
description: "Introducing Doks, a Hugo theme helping you build modern documentation websites that are secure, fast, and SEO-ready — by default."
excerpt: "Introducing Doks, a Hugo theme helping you build modern documentation websites that are secure, fast, and SEO-ready — by default."
date: 2020-11-04T09:19:42+01:00
lastmod: 2020-11-04T09:19:42+01:00
draft: false
weight: 50
images: []
categories: ["News"]
tags: ["security", "performance", "SEO"]
contributors: ["Henk Verlinde"]
pinned: false
homepage: false
---
